File: src/main/java/io/github/alexiscomete/lapinousecond/commands/classes/Give.java
package io.github.alexiscomete.lapinousecond.commands.classes;

import io.github.alexiscomete.lapinousecond.commands.CommandBot;
import org.javacord.api.event.message.MessageCreateEvent;

public class Give extends CommandBot {
    public Give() {
        super("description", "give", "totalDescription");
    }

    @Override
    public void execute(MessageCreateEvent messageCreateEvent, String content, String[] args) {

    }
}

File: src/main/java/io/github/alexiscomete/lapinousecond/commands/classes/Give.kt
package io.github.alexiscomete.lapinousecond.commands.classes

import io.github.alexiscomete.lapinousecond.commands.CommandBot
import org.javacord.api.event.message.MessageCreateEvent

class Give : CommandBot("description", "give", "totalDescription") {
    override fun execute(messageCreateEvent: MessageCreateEvent, content: String, args: Array<String>) {}
}